src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -21,6 +23,32 @@ public void setUp() throws Exception {
         }
     }
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+
+    @Test
+    public void testImportEntriesException() throws IOException {
+        thrown.expect(IOException.class);
+
+        CopacImporter importer = new CopacImporter();
+
+        @SuppressWarnings("unused")
+        List<BibtexEntry> entries = importer.importEntries(null, new OutputPrinterToNull());
```

Comment:
Can you please write this without an assignment? Then you do not need the `@SuppressWarnings` annotation anymore.